libs/R-TypeLogic/EntityManipulation/ButtonManipulation/Components/DisplayState.hpp

```diff
@@ -0,0 +1,32 @@
+/*
+** EPITECH PROJECT, 2022
+** Project
+** File description:
+** DisplayState
+*/
+
+#ifndef DISPLAYSTATE_HPP_
+#define DISPLAYSTATE_HPP_
+
+#include "Component/Component.hpp"
+#include "R-TypeLogic/EntityManipulation/ButtonManipulation/SharedResources/MenuStates.hpp"
+
+namespace ecs
+{
+
+    class DisplayState : public Component{
```

Comment:
does it respect the clang format ?

Reply:
Does it pass the CI ?

Reply:
It seems but doesn't need a space right after the `publi Component` ?